elementary/monitor/dbt_project/macros/get_resources_latest_invocation.sql

```diff
@@ -0,0 +1,22 @@
+{% macro get_resources_latest_invocation() %}
+  {% set dbt_run_results = ref('dbt_run_results') %}
+  {%- if elementary.relation_exists(dbt_run_results) -%}
+    {% set get_resources_latest_invocation_query %}
+      with row_numbered_run_results as (
```

Comment:
Lets give it a more meaningful name than row_numbered 🙂 
Something like `ordered_run_results` or something that explains what is the meaning of the row_number